auth0/resource_auth0_attack_protection.go

```diff
@@ -36,7 +37,9 @@ func newAttackProtection() *schema.Resource {
 							Description: "Whether or not breached password detection is active.",
 						},
 						"shields": {
-							Type: schema.TypeList,
+							Type:     schema.TypeSet,
```

Comment:
Changing these to a TypeSet so we don't constantly get diffs if the order of the shield differs, but not the content, as the API doesn't always return them in the same order.